spec/acceptance/exec_powershell_spec.rb

```diff
@@ -34,6 +34,31 @@
 
   end
 
+  describe 'should handle a try/catch successfully' do
```

Comment:
In my experience with this bug, I couldn't get anything in the `catch` block to run. If the `try` failed, the script would just silently continue on.

There should be a test that the `try` block works as well as the `catch` block.